peopleops/index.md

```diff
@@ -24,6 +24,16 @@ Sick leave, or having limited availibility is not recorded currently. Keep your
 manager updated on your health, and let them know what FlowForge can do for you
 to aid in your recovery.
 
+## Parental Leave
+
+If you live in a country where a statutory Parental Leave benefit is available,
+you will be required to claim statutory Parental Leave pay (if you are eligible).
+
+Any employee can take up to 6-weeks if they wish without requiring manager approval. We recommend a minimum of 4 weeks, but recognise it's a personal choice to make. Further leave can be requested, but should be discussed with your manager.
+
+If you live in a country that offers statuatory leave longer than this, then
```

Comment:
@ZJvandeWeg I tried to cover statutory maternity leave with this statement, for example, in the UK:
- 90% salary for the first 6 weeks (we improve on this with full pay)
- £151.97 or 90% of your average weekly earnings (whichever is lower) for the next 33 weeks (we could offer full pay for more weeks up front), but would otherwise offer statutory.